spyder/plugins/ipythonconsole/comms/tests/test_console_kernel.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug  5 16:43:33 2019
+
+@author: quentinpeter
+"""
```

Comment:
Please remove this header and use the one we use in all our other files instead.